src/MyPlot/MyPlot.php

```diff
@@ -615,7 +617,7 @@ public function clearPlot(Plot $plot, int $maxBlocksPerTick = 256) : bool {
 		}
 		$maxBlocksPerTick = $ev->getMaxBlocksPerTick();
 		foreach($this->getServer()->getLevelByName($plot->levelName)->getEntities() as $entity) {
-			if($this->getPlotBB($plot)->isVectorInXZ($entity)) {
+			if($this->getPlotBB($plot)->isVectorInXZ($entity) && $entity->getY() <= $this->getLevelSettings($plot->levelName)->groundHeight) {
```

Comment:
This change is unrelated to this PR

Reply:
This should be in the fillPlot function. I'll fix it.